File: lp1/lista1/15.py
n1 = int(input("Digite o primeiro número: "))
n2 = int(input("Digite o segundo número: "))

if n1 >= n2:
    maior = n1
    menor = n2
elif n2 > n1:
    maior = n2
    menor = n1

for i in range(menor + 1, maior):
    print(i)

File: lp1/lista1/15.c
/*
Solicitar vários valores inteiros e apos o usuário digitar qualquer valor
negativo, o programa deverá mostrar o maior e o menor valor digitado.
*/

#include <stdio.h>

int main()
{
    int num, maior, menor, contador;
    contador = 0;
    while (1)
    {
        printf("Digite um valor: (negativo p/ parar)\n");
        scanf("%d", &num);
        if (num >= 0)
        {
            if (contador == 0)
                maior = menor = num;
            else
            {
                if (num > maior)
                    maior = num;
                else if (num < menor)
                    menor = num;
            }
        }
        else
            break;
        contador++;
    }
    printf("O maior valor digitado foi %d e o menor foi %d\n", maior, menor);
    return 0;
}